lib/src/Base/Geom/Domain.cxx

```diff
@@ -71,6 +72,13 @@ Sample Domain::computeDistance(const Sample & sample) const
   return getImplementation()->computeDistance(sample);
 }
 
+/** Draw the sample as a cloud of points in or out of the domain */
+GridLayout Domain::draw(const Sample& sample,
+  const String& inColor,
```

Comment:
This method should be in visualtest instead

Reply:
done